adr/0005-code-formatting.md

```diff
@@ -0,0 +1,28 @@
+# 4. Code Formatting
+
+Date: 2019-07-07
+
+Architecture issue: [#249](https://github.com/home-assistant/architecture/issues/249)
+
+## Status
+
+Accepted
+
+## Context
+
+Proper code formatting is vital to the long-term viability and maintainability of a code
+base. In place, a codebase with such formatting is:
+
+- easier to review
+- easier to diff against previous commits
+- more likely to be free of linting errors
+
+## Proposal
+
+- We use [Black](https://github.com/python/black) as the de-facto code formatter for the entire back-end codebase
```

Comment:
```suggestion
- We use [Black](https://github.com/python/black) as code formatter for the back-end codebase
```

To me, de-facto and entire are weasel words and I do not think they are needed here